purchase_request/views/purchase_request_view.xml

```diff
@@ -194,6 +194,7 @@
                                         widget="analytic_distribution"
                                         groups="analytic.group_analytic_accounting"
                                         options="{'product_field': 'product_id', 'business_domain': 'purchase_order'}"
+                                        optional="hide"
```

Comment:
I'm seeing that in invoices and other documents, the field can be hidden, but it's shown by default.

```suggestion
                                        optional="show"
```

Reply:
Already exists in the tree view of the lines `optional=hide` (https://github.com/OCA/purchase-workflow/blob/b4670f4f387c96648a8609111a6758f71d5e595d/purchase_request/views/purchase_request_line_view.xml#L46), and in `purchase` too https://github.com/odoo/odoo/blob/1d605fe49aa3a6ed45943312627f0d56b1fb7aee/addons/purchase/views/purchase_views.xml#L241

Reply:
OK, let's go with that approach.